webapp/packages/plugin-data-spreadsheet-new/src/DataGrid/Actions/Editing/ACTION_DATA_GRID_EDITING_SET_TO_NULL.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { createAction } from '@cloudbeaver/core-view';
+
+export const ACTION_DATA_GRID_EDITING_SET_TO_NULL = createAction('data-grid-editing-set-to-null', {
+  label: 'data_grid_table_editing_set_to_null',
+});
```

Comment:
I am not sure that we need to use action for each of the null or ordering menu items; you can use the base menu item instead.
Action entity is more about global things that can be used together with shortcuts, so we don't use them for generated or loaded data so as for multiple options (like dark/light or English, Russian etc)